src/extensions/react-flow/nodes/NodeContent.tsx

```diff
@@ -341,11 +349,15 @@ export function NodeContent<CONTENT_PROPS = any>({
 
     const { handles = defaultHandles(flowVersionCheck), ...otherProps } = otherDomProps;
 
-    const isResizeable = !!onNodeResize && minimalShape === "none";
+    const isResizable = !!onNodeResize && minimalShape === "none";
```

Comment:
Misses test on correct condition for `resizeDirections`, currently the resize container is also used for a value of `{}` or `false` for every direction.